src/main/kotlin/net/atos/zac/app/zaken/converter/historie/RESTZaakHistoriePartialUpdateConverter.kt

```diff
@@ -50,6 +52,17 @@ class RESTZaakHistoriePartialUpdateConverter @Inject constructor(
                     .let(vrlClientService::findCommunicatiekanaal)
                     .map(CommunicatieKanaal::getNaam)
                     .orElse(null)
+            resource == "hoofdzaak" && item is String ->
```

Comment:
I would make "hoofdzaak" a constant, if it is not already?

Reply:
Not part of this PR maybe but I think we can change:

```private fun convertValue(resource: Any?, item: Any?): String? ```

to:

```private fun convertValue(resource: Any, item: Any): String```

because I would expect these to be non-nullable?

Reply:
Good point about the constant, I forgot to do that.
About the other comment: I managed to do it, but it requires some more code in the calling method. Let me know what you think.